src/lang/ukrainian.txt

```diff
@@ -1144,6 +1144,7 @@ STR_GAME_OPTIONS_CURRENCY_IDR                                   :Індонез
 STR_GAME_OPTIONS_CURRENCY_MYR                                   :Малайзійський рингіт
 STR_GAME_OPTIONS_CURRENCY_LVL                                   :Латвійські лати
 STR_GAME_OPTIONS_CURRENCY_PTE                                   :Португальське ескудо
+STR_GAME_OPTIONS_CURRENCY_UAH                                   :Українська гривня
```

Comment:
When adding strings to OpenTTD, we only change `english.txt` in the PR, and let Web Translator handle all other languages. 🙂